### PR TITLE
Assure that TICK is different for multi-textured renderers

### LINKED ISSUE #5520: PIXI.Graphics initially drawing transparent
### Expected Behavior

I'm trying to draw a white bar attached to the player sprite.

### Current Behavior

The bar is drawn transparent initially. You can see the area blocked off in the upper right here: https://imgur.com/5gsTd06 Oddly enough, after firing the ship's weapon a few times which essentially creates a few more sprites that quickly disappear the graphics turn white and stay that way. As in: https://imgur.com/gI9ilpF This is strange because I'm completely clearing the stage and recreating everything every frame.

### Steps to Reproduce

Here's my code:

```typescript
const createCapacitorBar = (capacitor: Capacitor) => {
  const bar = new PIXI.Graphics();
  bar.beginFill(0xffffff);
  bar.drawRect(0, 0, 100, 100);
  bar.endFill();
  return bar;
};
```

And elsewhere:

```typescript
    // add player
    const playerSprite = createSprite(player.texture, player.scale, player.static);
    playerSprite.addChild(createCapacitorBar(player.capacitor));
    this.stage.addChild(playerSprite);
    this.stage.addChild(createCollisionShape(player));
```

### Environment

- **`pixi.js` version**: v5.0.0-rc.2
- **Browser & Version**: Chrome 73
- **OS & Version**: Ubuntu 18.04.2

### PR DESCRIPTION
**Pull request by ivanpopelyshev**
Fixes #5520 

I added methods to BaseTexture because that's where `_batchEnabled` field is stored.

I also suggest that we expose those methods for plugin-makers but `_` means that it can be changed in any version without deprecation.

Broken: https://www.pixiplayground.com/#/edit/1WCb9fBbUQieUK2IdMKzR
Works: https://www.pixiplayground.com/#/edit/69HLoVsdpfl0NBwEyJws0

### CODE REVIEW

**Comment by ivanpopelyshev** [comment]
@btmorex 

Thanks, I stumbled across this problem when i was making plugins for v4.

File: packages/core/src/textures/BaseTexture.js [hunk 1]
```diff
@@ -632,4 +639,29 @@ export default class BaseTexture extends EventEmitter
...
+    /**
+     * Called from multi-texture rendered to save the last batch number.
+     * Only first 32 bits matter.
+     *
+     * @static
+     * @param {number} savedValue last used batch number
+     */
+    static _batchEnd(savedValue)
+    {
+        BaseTexture._globalBatch = savedValue | 0;
```
Comment (GoodBoyDigital) [inline packages/core/src/textures/BaseTexture.js]: looking good, can you explain how this magic line works pls? Why only first 32 bits matter? Just curious! :)

Comment (ivanpopelyshev) [reply packages/core/src/textures/BaseTexture.js]: just in case we have more than 2^54 texture batches. Ugh... 100 years I think? lets remove that.

Comment (ivanpopelyshev) [reply packages/core/src/textures/BaseTexture.js]: ok, changed.

**Comment by bigtimebuddy** [comment]
@GoodBoyDigital did you have any other feedback on this from last we talked?

File: packages/graphics/src/GraphicsGeometry.js [hunk 2]
```diff
@@ -580,7 +579,7 @@ export default class GraphicsGeometry extends BatchGeometry
      */
     buildDrawCalls()
     {
-        TICK++;
+        let TICK = BaseTexture._batchBegin();
```
Comment (GoodBoyDigital) [inline packages/graphics/src/GraphicsGeometry.js]: how about 

```let TICK = BaseTexture._globalBatch++;```

File: packages/graphics/src/GraphicsGeometry.js [hunk 3]
```diff
@@ -678,6 +677,8 @@ export default class GraphicsGeometry extends BatchGeometry
             this.addTextureIds(textureIds, textureId, data.attribSize);
         }
 
+        BaseTexture._batchEnd(TICK);
```
Comment (GoodBoyDigital) [inline packages/graphics/src/GraphicsGeometry.js]: and ```BaseTexture._globalBatch = TICK;```

**Pull request merged**